buildpacks/nodejs-function-invoker/shpec/build_shpec.sh

```diff
@@ -19,6 +19,20 @@ create_temp_app_dir() {
 }
 
 describe "lib/build.sh"
+	describe "run_initial_checks"
```

Comment:
I think it would be worth also testing the "working" case, plus also the "package.json doesn't have a 'main' key" case.